lib/gooddata/lcm/actions/synchronize_etls_in_segment.rb

```diff
@@ -81,13 +81,42 @@ def call(params)
             end
           end
 
```

Comment:
The integration is going to test the happy path. It would be nice to write unit tests for the corner cases.